packages/api-request-builder/src/delete.js

```diff
@@ -0,0 +1,16 @@
+/* @flow */
+/**
+ * Set the `version` number to the internal state of the service instance
+ * in order to generate a delete uri
+ *
+ * @param  {number} version - The version of the item
+ * @throws if `version` is missing or not a number.
+ * @return {Object} The instance of the service, can be chained.
+ */
+
+export default function toDelete (version: number): Object {
+  if (!version || typeof version !== 'number')
+    throw new Error('Required version to delete missing or not valid')
```

Comment:
Let's use this

`A resource version is invalid or missing` or something like this. This explains more what *version* is and easily connected to the platform definition of version